### ISSUE TITLE
Feature: Share link to external site

### ISSUE THREAD

**Issue by hbt-dk** [owncloud/core#20560]
One of our customer wants to share a link to an external site. He has to create a file and put the link in it, but its worthy to open it and copy the link out of it. Is there any way to create a simple link and clicking on it will redirect to the url then? It would be a nice feature... Maybe use .url files for such feature?

**Comment by MorrisJobke** [comment]
Can you explain more detailed what steps aren't possible nowadays and how you want it to behave?

**Comment by hbt-dk** [comment]
The user has to create a file containing a url, e.g. "http://github.com/owncloud/", upload it and then shares the file with another user or group. The end users has to copy the url out of it to view the page. My first idea was to upload a file (e.g. github.url), and when the user clicks on it, the url will be opened directly. 

I found a Module named Shorty: https://apps.owncloud.com/content/show.php?content=150401

This Module is almost what i'm looking for, but it is stated as "This app should NOT be installed or activated.", and i dont need short urls. A simple list with urls (targets) which can be shared easily. This would be a nice tiny feature i think.

**Comment by MorrisJobke** [comment]
> This Module is almost what i'm looking for, but it is stated as "This app should NOT be installed or activated.", and i dont need short urls. A simple list with urls (targets) which can be shared easily. This would be a nice tiny feature i think.

So you want to share a URL by sharing a URL and not want to have it shortened? Simply share the original URL. I don't see the point in have another indirection.

**Comment by hbt-dk** [comment]
But how can i share a link with others without creating a new file or folder for it? Lets say i want to share the link http://github.com/owncloud to users and groups in owncloud, how do i do that? I am not able to modify the text field beneath the **Share link** option.

There is a option under the **New** menue named **From link**, but this downloads the resource behind the link. This is also a nice feature, but in some cases i dont want do download the file and only want to send the link to others, like download links for actual software that the users have to install (for example).

It would be really cool to have an option **External Link** which creates a **.url** file or something. When sharing this one the users get either the External Link by mail or the link to the **.url** file instead, which will redirect if opened. The users are then also able to click on the **.url** file which will directly open the link saved in the file in a new window.

The gui of the module Shorty is really nice. It does what i want to do, but i dont need an own gui for that, and the shortener is not really required (would also be a nice feature for owncloud i think, but not recommended). 

In short: It would be nice to tweak owncloud a bit to handle **.url** files for sharing links to external pages.

**Comment by MorrisJobke** [comment]
I don't think that this will be build into the core product. This is mostly what an app can cover.

**Comment by MorrisJobke** [comment]
As this is a feature request that best should be handled as an separate app I would like to ask you to reopen this in the apps repository: http://github.com/owncloud/apps/issues

Thanks
